### DISCUSSION TITLE
What happens if Gnome/KDE drop X11 session?

### DISCUSSION THREAD

**Discussion by HubKing** [neutrinolabs/xrdp#2840] [Q&A]
XRDP seems to be using X11 session, and I have read that both Gnome and KDE are considering dropping X11 session in the upcoming versions (maybe it would take a few years). Will XRDP still work, if the DE I am using does not support X11 session?

**Comment by matt335672** [comment]
This is effectively a duplicate of #2637. I'll comment further there.
